lib/broadside/deploy/ecs_deploy.rb

```diff
@@ -73,6 +73,20 @@ def bootstrap
         info "Service '#{family}' doesn't exist, creating..."
         EcsManager.create_service(config.ecs.cluster, family, @deploy_config.service_config)
       end
+
+      run_bootstrap
+    end
+
+    def run_bootstrap
```

Comment:
I thought we didn't need to run `update_task_revision` before the bootstrap commands because the task definition has already been created?